### DISCUSSION TITLE
Suggestion: Play "Telemetry Recovered" sound first time telemetry is received

### DISCUSSION THREAD

**Discussion by zvonler** [EdgeTX/edgetx#4210] [General]
The current behavior of audio telemetry warnings is to play AU_TELEMETRY_LOST when telemetry is lost, and to play AU_TELEMETRY_BACK when it is restored. I would like EdgeTX to play AU_TELEMETRY_BACK also when transitioning from `telemetryState == TELEMETRY_INIT`, so that there is always audio confirmation that telemetry has been received. Right now, after first powering up, there is no audio confirmation that telemetry is working.

I think a reasonable way to get that behavior would be to change the check here https://github.com/EdgeTX/edgetx/blob/ee9d50dca4eb4fa0e6b485966c3e23551102cbd5/radio/src/telemetry/telemetry.cpp#L353

to be `if (telemetryState != TELEMETRY_OK)`.

**Comment by pfeerick** [comment]
IMO that is invalid, as it is not recovered (since that implies it was lost), but initialised/connected/established/online (pick a word! 🤣) ... so perhaps it would be better to have its own phrase (which also gives you the option of stopping that behaviour if you don't want it)?

**Comment by zvonler** [comment]
I agree adding a new sound would be even better, and can imagine some people would want to be able to turn it off.

**Reply by pfeerick** [reply]
What I will do then is add the same as the trainer... as that recently got the TRAINER_CONNECTED event added... 

```
#define AUDIO_TRAINER_CONNECTED() audioEvent(AU_TRAINER_CONNECTED)
#define AUDIO_TRAINER_LOST()     audioEvent(AU_TRAINER_LOST)
#define AUDIO_TRAINER_BACK()     audioEvent(AU_TRAINER_BACK)
```

Telemetry currently only has
```
#define AUDIO_TELEMETRY_LOST()   audioEvent(AU_TELEMETRY_LOST)
#define AUDIO_TELEMETRY_BACK()   audioEvent(AU_TELEMETRY_BACK)
```

**Comment by 3djc** [comment]
Oh yes, I will definitely not enable that sounds. Alerts should be for errors, not when everything works as expected
